container-aws-csharp/Pulumi.yaml

```diff
@@ -0,0 +1,18 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: yaml
+template:
+  description: A containerized service on AWS Fargate
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
+    containerPort:
+      description: The port to expose on the container
+      default: 80
```

Comment:
Out of curiosity, is there a reason these values aren't explicitly in quotes like for the YAML in the other languages?